jwala-services/src/main/java/com/cerner/jwala/service/custom/logging/WinSvcPasswordMaskingLayout.java

```diff
@@ -0,0 +1,89 @@
+package com.cerner.jwala.service.custom.logging;
+
+import org.apache.commons.lang3.StringUtils;
+import org.apache.log4j.Level;
+import org.apache.log4j.Logger;
+import org.apache.log4j.PatternLayout;
+import org.apache.log4j.spi.LoggingEvent;
+
+import java.util.*;
+
+/**
+ * Masks the Windows Service Password (svc_password)
+ * Created by Jedd Cuison on 6/6/2017
+ */
+public class WinSvcPasswordMaskingLayout extends PatternLayout {
+
+    private static final String INSTALL_SERVICE_SH = "install-service.sh";
+    private static final String SET_SVC_PASSWORD = "set svc_password=";
```

Comment:
Is it possible to create the above 2 constants as parameters in the log4j.xml?

I'm just thinking that these searches are very specific to the current scripts, and in case it changes in the future it would be easier to just update the log4j.xml then have to rebuild the whole project.